packages/train2go-bridge/test/background.test.js

```
@@ -242,6 +242,35 @@ describe("background service worker", () => {
       ).rejects.toThrow("Missing userId");
     });
 
+    it("read-day backfills the date param onto every parsed activity (the daily HTML fragment lacks a date anchor)", async () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)